### PR TITLE
refactor(docs): css update

### PR DESCRIPTION
**Pull request by yinonov**

### CODE REVIEW

File: apps/docs/transformers/code-block-demo.js [hunk 1]
```diff
@@ -117,11 +117,9 @@ const style = `
 	overflow: hidden;
 }
 .${CBD_DEMO} {
-	padding: 20px;
```
Comment (rachelbt) [inline apps/docs/transformers/code-block-demo.js:120]: the padding removal is needed UI-wisely :)
I get that it breaks the design so we can add:
.${CBD_DEMO} iframe {
margin: 20px;
}

Comment (yinonov) [reply apps/docs/transformers/code-block-demo.js:120]: it doesn't look good in some demos. like if there's shadow it just cuts it. the inner body should probably have padding

Comment (rachelbt) [reply apps/docs/transformers/code-block-demo.js:120]: I guess that the shadow is also doesn't look complete when the elements are aligned to the top & right ;)

File: apps/docs/transformers/code-block-demo.js [hunk 2]
```diff
@@ -12,9 +12,9 @@ const CBD_DETAILS = 'cbd-details';
 const CBD_BUTTON_SHOW = 'cbd-button-show';
 const CBD_CODE_BLOCK = 'cbd-code-block';
 
-const MAIN_STYLE = '<link rel="stylesheet" href="/assets/styles/main.css">';
+const MAIN_STYLE = '<link rel="stylesheet" href="/assets/styles/iframe.css">';
```
Comment (rachelbt) [inline apps/docs/transformers/code-block-demo.js:15]: 👍

**Review by rachelbt — DISMISSED** [review]
now the preview looks great!! :)

**Review by rachelbt — APPROVED** [review]

**Pull request merged**